deepcell/image_generators/semantic.py

```diff
@@ -367,10 +375,11 @@ def random_transform(self, x, y=None, seed=None):
         """
         params = self.get_random_transform(x.shape, seed)
 
+        # temporarily convert to float32 so enable scipy.nd_image functions to work
```

Comment:
We say temporarily, but do we cast it back?

Reply:
No, I guess it's not actually temporary, just on a per-batch basis which has a small memory footprint